frontend/src/pages/AboutPage.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import withLayout from './withLayout'
+
+const About = () => (
+  <div className="center-narrow">
+    <h1>Welcome</h1>
+    <p>Welcome to the new Penn Course Review!</p>
```

Comment:
Kinda odd to have two welcomes...maybe make the first one "Hey there!" or something